treeherder/seta/tasks.py

```diff
@@ -0,0 +1,15 @@
+from cerely import task
+from treeherder.seta.analyze_failures import AnalyzeFailures
+from treeherder.seta.preseed import load_preseed
+
+
+@task(name='seta-analyze-failures', time_limit=20*60)
+def seta_analyze_failures():
+    '''We analyze the starred test failures of the last 90 days which were fixed by a commit.'''
+    AnalyzeFailures().run()
```

Comment:
is the time limit set to 20 minutes?  That seems pretty high- it should be <5 minutes without optimizations.